app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class PasswordsController < Devise::PasswordsController
+      include API::V1::OauthApplicationProtection
+
+      def create
+        super do |user|
+          if user.valid?
+            render json: {
```

Comment:
Just noticed what, to align with the JSON API spec, we must send `id` and `type`. If we don't have `id`, I think we can generate a random ID just to align with JSON API spec. What do you think?

Reply:
@junan Good catch! However, from the document top level (https://jsonapi.org/format/#document-top-level), we don't need to contain `data - the document’s “primary data”`. In this case, I just need to use the `meta` object which is not required `id` and `type`. What do you think?